node_modules/oae-tenants/tests/test-tenants.js

```diff
@@ -325,6 +325,21 @@ describe('Tenants', function() {
 
             callback();
         });
+
+        /**
+         * Test that verifies that the email domain index can map multiple email domains to an alias
+         */
+        it('verify it supports multiple tenants', function(callback) {
+            var index = _createIndex();
+
+            index.update('sussex', 'sussex.ac.uk');
+            index.update('sussex', 'susx.ac.uk');
+
+            assert.strictEqual(index.match('sussex.ac.uk'), 'sussex');
+            assert.strictEqual(index.match('susx.ac.uk'), 'sussex');
+
+            callback();
+        });
```

Comment:
I'd feel a bit safer with more tests based around adding, deleting and checking for conflicts for test cases with multiple email domains.